charts/opentelemetry-operator/values.yaml

```diff
@@ -360,6 +360,9 @@ nodeSelector:
 topologySpreadConstraints: []
 hostNetwork: false
 
+# Whether to use user namespace or not
```

Comment:
Please add a comment explaining this is only available on k8s 1.33 and above.

Reply:
added